sdk/digitaltwins/digitaltwins/src/digitalTwinsClient.ts

```diff
@@ -0,0 +1,929 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
See suggested changes in this commit: 

https://github.com/xirzec/azure-sdk-for-js/commit/a03822e7d5063490b3572b5d4283b8208df5464d

Or if you want to copy/paste the whole file over this one in your local:

https://raw.githubusercontent.com/xirzec/azure-sdk-for-js/a03822e7d5063490b3572b5d4283b8208df5464d/sdk/digitaltwins/digitaltwins/src/digitalTwinsClient.ts

Reply:
Thanks a lot Jeff! I have accepted and checked in your suggestions.

---
In reply to: [475982304](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r475982304) [](ancestors = 475982304)